test/libsolidity/SolidityExpressionCompiler.cpp

```diff
@@ -656,6 +656,23 @@ BOOST_AUTO_TEST_CASE(blockhash)
 	BOOST_CHECK_EQUAL_COLLECTIONS(code.begin(), code.end(), expectation.begin(), expectation.end());
 }
 
+BOOST_AUTO_TEST_CASE(blobhash)
+{
+	char const* sourceCode = R"(
+		contract test {
+			function f() public {
+				blobhash(3);
+			}
+		}
+	)";
+
+	bytes code = compileFirstExpression(sourceCode, {}, {});
+
+	bytes expectation({uint8_t(Instruction::PUSH1), 0x03,
+					   uint8_t(Instruction::BLOBHASH)});
```

Comment:
```suggestion
	bytes expectation({
		uint8_t(Instruction::PUSH1), 0x03,
		uint8_t(Instruction::BLOBHASH)
	});
```

Also, the test is failing:
```
/solidity/test/libsolidity/SolidityExpressionCompiler.cpp(132): fatal error: in "SolidityExpressionCompiler/blobhash": Resolving names failed
```
Probably because the test is not being skipped before Cancun.